### ISSUE TITLE
CMS 3.2.x, structure mode : Clipboard is never visible

### ISSUE THREAD

**Issue by DylannCordel** [divio/django-cms#5019]
In structure mode, when we copy or cut a plugin, page reloads with a success message but clipboard stay hidden but is in HTML (see code bellow) :
Furthermore, "Paste" action in placeholders stays inactive.

Tested with 3.2.0 and 3.2.1. Maybe related to #5018

```html
<div class="cms-clipboard" data-title="Presse-papiers"><div style="display: block; opacity: 1;" class="cms-clipboard-containers cms-dragarea cms-draggables ui-sortable">
<div class="cms-draggable cms-draggable-17      cms-draggable-disabled      cms-draggable-from-clipboard ui-sortable-handle">
    <div class="cms-dragitem cms-dragitem-handler">
            <div class="cms-submenu-btn cms-submenu-add cms-btn cms-btn-disabled">   
                <span class="cms-hover-tooltip" data-cms-tooltip="You cannot add plugins to this plugin."></span>
            </div>
            <div class="cms-submenu-btn cms-submenu-edit cms-btn" data-rel="edit">
                <span class="cms-hover-tooltip cms-hover-tooltip-left cms-hover-tooltip-delay" data-cms-tooltip="Modifier"></span>
            </div>
            <div class="cms-submenu cms-submenu-settings cms-submenu-btn cms-btn"></div>
        <div class="cms-submenu-dropdown cms-submenu-dropdown-settings">
            <div class="cms-dropdown-inner">
                <div class="cms-submenu-item"><a data-cms-icon="scissors" data-rel="cut" href="#">Couper</a></div>
                <div class="cms-submenu-item"><a data-cms-icon="copy" data-rel="copy" href="#">Copier</a></div>
                <div class="cms-submenu-item cms-submenu-item-disabled">
                    <a data-cms-icon="paste" data-rel="paste" href="#">coller</a>
                    <span style="display: block;" class="cms-submenu-item-paste-tooltip cms-submenu-item-paste-tooltip-empty cms-hover-tooltip cms-hover-tooltip-left cms-hover-tooltip-delay" data-cms-tooltip="Clipboard is empty."></span>
                    <span class="cms-submenu-item-paste-tooltip cms-submenu-item-paste-tooltip-restricted cms-hover-tooltip cms-hover-tooltip-left cms-hover-tooltip-delay" data-cms-tooltip="This plugin does not allow plugins of this type as nested plugins."></span>
                    <span class="cms-submenu-item-paste-tooltip cms-submenu-item-paste-tooltip-disabled cms-hover-tooltip cms-hover-tooltip-left cms-hover-tooltip-delay" data-cms-tooltip="This plugin cannot have nested plugins."></span>
                </div>
                <div class="cms-submenu-item"><a data-cms-icon="bin" data-rel="delete" href="#">Supprimer</a></div>
<div class="cms-submenu-item"><a href="/admin/cms/page/plugin/alias_plugin/create_alias/" data-rel="ajax" data-post="{&quot;plugin_id&quot;: 17, &quot;csrfmiddlewaretoken&quot;: &quot;iUNUWDivSIyx5TgcMQ8BAuIiubA3CIEP&quot;}" data-on-success="REFRESH_PAGE">Créer alias</a></div>
            </div>
        </div>
        <span class="cms-submenu-lang">fr</span>
        <span class="cms-dragitem-text" title="TextPlugin ID: 17"><strong>Texte</strong> Ceci est le...</span>
    </div>
    <div class="cms-collapsable-container cms-hidden">
    </div>
</div>
    </div>
    <div class="cms-plugins">
        <div class="TextPlugin cms_plugin cms_plugin-17"></div>
    </div>
</div>
```

Tested with 3.2.0 and 3.2.1.

**Comment by vxsx** [comment]
Clipboard is not visible by default, but can be opened from the menu

![screen shot 2016-02-18 at 15 03 11](https://cloud.githubusercontent.com/assets/366813/13145416/c47a247a-d650-11e5-83c7-9dfc247ab8be.png)

Paste can be disabled for multiple reasons:
- clipboard is empty
- clipboard contains a plugin that cannot have a parent of the type, that is the given plugin/placeholder
- clipboard contains a plugin that cannot be a child of the given plugin/placeholder

If you hover over disabled paste button for several seconds you should see the tooltip that explains the problem:

![screen shot 2016-02-18 at 15 05 49](https://cloud.githubusercontent.com/assets/366813/13145464/23c1a174-d651-11e5-9cb8-95ad56fc8f35.png)

**Comment by DylannCordel** [comment]
Thx @vxsx 
I closes this one, it's the same "problem" than #5018
